### PR TITLE
Remove deprecated code.

### PR DESCRIPTION
**Pull request by ioquatix**

### CODE REVIEW

File: test/spec_utils.rb [hunk 1]
```diff
@@ -702,17 +658,10 @@ def initialize(*)
...
-    Rack::Utils.make_delete_cookie_header(nil, 'name', {}).
-      must_equal "name=; max-age=0; expires=Thu, 01 Jan 1970 00:00:00 GMT"
-    Rack::Utils.add_remove_cookie_to_header(nil, 'name').
-      must_equal "name=; max-age=0; expires=Thu, 01 Jan 1970 00:00:00 GMT"
-  end
 end
 
 describe Rack::Utils, "get_byte_ranges" do
-  deprecated "pase simple byte ranges from env" do
+  it "pase simple byte ranges from env" do
```
Comment (jeremyevans) [inline test/spec_utils.rb:664]: Not a bug in the PR, but may as well fix the typo in the description: `s/pase/parse/`

**Review by jeremyevans — APPROVED** [review]
Looks good.  See one inline comment, but that can be in a separate PR if you want.

File: test/spec_utils.rb (hunk 1 above)
Comment (ioquatix) [reply test/spec_utils.rb:664]:
```suggestion
  it "parse simple byte ranges from env" do
```

**Review by cklim24 — COMMENTED** [review]
gem install sinatra
gem install puma # or any other server

**Pull request merged**